src/i18n/en.js

```diff
@@ -225,6 +225,7 @@ export default {
     error_invalid_domain_to_register_account: `Ouch! Invalid Email to create an account.`,
     error_invalid_email_address: `Ouch! Enter a valid Email.`,
     error_invalid_login: `Ouch! There is an error in your Username or Password. Please, try again.`,
+    error_invalid_name: `Ouch! Enter a valid name.`,
```

Comment:
If you use `Nombre` with caps, I think in english should be the same: `Name` right?

Reply:
fixed